calico/manifest.json

```
@@ -0,0 +1,36 @@
+{
+  "display_name": "Calico",
+  "maintainer": "max@galadrim.fr",
+  "manifest_version": "1.0.0",
+  "name": "calico",
+  "metric_prefix": "calico.",
+  "metric_to_check": "calico.felix_active_local_endpoints",
```

```suggestion
  "metric_to_check": "calico.felix.active.local_endpoints",
```
If you choose to use the pseudo-hierarchical  format suggested in the metrics.py, this requires updating